Gemfile.lock

```diff
@@ -696,5 +703,8 @@ DEPENDENCIES
   wunderground (~> 1.2.0)
   xmpp4r (~> 0.5.6)
 
+RUBY VERSION
```

Comment:
Does anyone know if committing this has bad effects for different Ruby versions?